troubleshooting/index.md

```diff
@@ -7,6 +7,7 @@ This document provides troubleshooting information for services and functionalit
     * [Hardware Discovery](#known-issues-hardware-discovery)
     * [initrd.img.xz not found](#initrd-not-found)
     * [SAT/HSM/CAPMC Component Power State Mismatch](known_issues/component_power_state_mismatch.md)
+    * [BOS/BOA Incorrect command is output to rerun a failed operation.](kwon_issues/incorrect_output_for_bos_command_rerun.md)
```

Comment:
kwon_issues > known_issues

Reply:
Are we missing the known issue doc?  Also typo in kwon_issues

Reply:
No, the know_issues document is the other file changed in this commit. I have also corrected the typo.